recipes-kernel/linux/linux-compulab-common.inc

```diff
@@ -21,3 +21,9 @@ BALENA_CONFIGS[caam] = " \
     CONFIG_CRYPTO_DEV_FSL_CAAM_JR=y \
     CONFIG_CRYPTO_DEV_FSL_CAAM_INTC=y \
 "
+
+BALENA_CONFIGS:append = " panic"
+BALENA_CONFIGS[panic] = " \
+    CONFIG_PANIC_ON_OOPS=y \
+    CONFIG_PANIC_TIMEOUT=1 \
```

Comment:
Wouldn't this be better suited in meta-balena? As far as I can tell this behaves relatively randomly between device types, but I think we should just universally have `PANIC_ON_OOPS=y`, and maybe use a higher timeout with `OS_DEVELOPMENT=1`.

Reply:
This is now in https://github.com/balena-os/meta-balena/pull/3582/files